metadata-ingestion/tests/unit/utilities/test_prefix_patch_builder.py

```diff
@@ -35,7 +35,7 @@ def test_build_prefix_batches_exceeds_max_batch_size():
     ]
     expected = [
         [PrefixGroup(prefix="app", names=["app"], exact_match=True)],
-        [PrefixGroup(prefix="app", names=["apple", "applet", "application"])],
+        [PrefixGroup(prefix="appl", names=["apple", "applet", "application"])],
```

Comment:
?

Reply:
related to the change in `metadata-ingestion/src/datahub/utilities/prefix_batch_builder.py` - fixing a subtle bug